src/implementation/Server/HTTPServer/actor.ts

```diff
@@ -39,10 +39,7 @@ export default class HTTPServerActor implements IServerActor {
    * This will create the routes that get invoked by the Dapr Sidecar
    */
   async init(): Promise<void> {
-    // Change the server startupDelay to 3s since we want the placement tables to get updated correctly
-    // @todo: is there a better way to detect this?
-    this.server.serverStartupDelay = 4000;
-    console.log("[Actors] Initializing, changing startup delay to 4s to allow for placement tables to propagate");
+    console.log("[Actors] Initializing");
```

Comment:
Should we remove the TODO?

Reply:
Also, will this work fine without the startup delay here?

Reply:
Yes, this can go as we use the health check this time. For placement tables we might need to check, but it seems to work as the tests are passing?

Reply:
True!